doc/source/_static/iiasa.css

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Just a thought, at some point we may want to make this a package, like `sphinx_rtd_theme`. That way we won't have to remember to find and sync CSS across multiple repos.

Reply:
+1, but let's save that for another day...